### ISSUE TITLE
Implement e-mail notification once the processing is done

### ISSUE THREAD

**Issue by JinIgarashi** [UNDP-Data/geohub-data-pipeline#19]
It would be nice to have email notification for the completion of data ingesting.

I just found the following article about Azure Communication Service.

https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/email/send-email?tabs=windows%2Cconnection-string&pivots=programming-language-python

I found Communication Service in Azure.

<img width="807" alt="image" src="https://github.com/UNDP-Data/geohub-data-pipeline/assets/2639701/8ea845d4-6831-445d-9f61-b761288f0529">

┆Issue is synchronized with this [Wrike task](https://www.wrike.com/open.htm?id=1206241805) by [Unito](https://www.unito.io)

**Comment by JinIgarashi** [comment]
Let me close this issue now since we failed to introduce Azure Communication Service because of IT blocking us
